src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-intro.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: External services
+metaDescription: "This is an introduction to the external services feature"
+redirects:
+  - /docs/apm/apm-ui-pages/monitoring/external-services-page-view-web-cloud-network-data/
+---
+The external services feature allows you to look closely at the upstream and downstream activity of a single service. Those upstream or downstream external services may be your own services you instrumented, or they may refer to services you simply call in the course of a transaction. 
+
+The external services feature is no longer just for New Relic APM agent users. It has expanded to include OpenTelemetry and now uses the power of distributed tracing to give you insights into service activity. The classic external services feature is still available behind a UI toggle, but we recommend you try out the full power the expanded external services.  
```

Comment:
Maybe change "no longer" to "not" so this doc doesn't reflect a particular point in time? Also maybe "It also includes OpenTelemetry and uses the power" ? Also, does "OpenTelemetry" in this sentence need an xref to a general OpenTelemetry doc so people know what that means in New Relic land?

Reply:
@barbnewrelic, I followed your suggestions!